tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bookmark_menu.py

```diff
@@ -20,7 +20,7 @@ class BookmarkMenu(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
```

Comment:
This is not going to work, because the class is called "BookmarkMenu" here.
You need to add a manifest_url thing here.